siteorigin-panels.php

```diff
@@ -1365,35 +1375,6 @@ function siteorigin_panels_render_form($widget, $instance = array(), $raw = fals
 }
 
 /**
- * This takes existing Page Builder data and makes it RTL by reversing the content
- */
-function siteorigin_panels_make_rtl($panels_data){
```

Comment:
Ah, ok, I see you removed `siteorigin_panels_make_rtl`. so you can ignore the previous comment about the added filter.